hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -65,40 +68,106 @@
     name: kubernetes-e2e-gce-master
     trigger-job: 'kubernetes-build'
     test-owner: 'Build Cop'
-    branch: 'master'
+    provider-env: '{gce-provider-env}'
     suffix:
         - 'gce':
-            description: 'Run E2E tests on GCE using the latest successful build.'
+            description: 'Runs all non-slow, non-serial, non-flaky, tests on GCE in parallel.'
             timeout: 30
+            job-env: |
+                # This is the *only* job that should publish the last green version.
+                export E2E_PUBLISH_GREEN_VERSION="true"
+                # This list should match the list in kubernetes-pull-build-test-e2e-gce.
+                export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+                export GINKGO_PARALLEL="y"
+                export PROJECT="k8s-jkns-e2e-gce"
         - 'gce-slow':
-            description: 'Run slow E2E tests on GCE using the latest successful build.'
+            description: 'Runs slow tests on GCE, sequentially.'
             timeout: 60
+            job-env: |
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Slow\] \
+                                         --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+                export GINKGO_PARALLEL="y"
+                export PROJECT="k8s-jkns-e2e-gce-slow"
         - 'gce-serial':
-            description: 'Run [Serial], [Disruptive], and [Feature:Restart] tests on GCE using the latest successful build.'
+            description: 'Run [Serial], [Disruptive], and [Feature:Restart] tests on GCE.'
```

Comment:
This description is wrong; we don't run feature:restart tests (in fact, I'm not sure that feature exists anymore as its own thing).